python/pyarrow/tests/test_types.py

```diff
@@ -94,39 +93,39 @@ def get_many_types():
 
 
 def test_is_boolean():
-    assert types.is_boolean(pa.bool_())
-    assert not types.is_boolean(pa.int8())
+    assert pa.types.is_boolean(pa.bool_())
```

Comment:
Are these changes necessary for some reason? They are not problematic, but they don't seem required either.

Reply:
You are right. I replaced `pa.types` to `types.`